waspc/examples/todoApp/src/client/pages/auth/AuthForm.jsx

```diff
@@ -0,0 +1,154 @@
+
+export default function AuthForm() {
+  return (
+    <>
+      {/*
```

Comment:
Which template does this comment refer to? The user doesn't have access to `index.html` in Wasp